libs/blocks/merch-card/merch-card.js

```diff
@@ -214,15 +214,17 @@ const parseContent = async (el, merchCard) => {
 };
 
 const getBadgeStyle = (badgeMetadata) => {
-  const badgeStyleRegex = /^#[0-9a-fA-F]+, #[0-9a-fA-F]+$/;
+  const badgeStyleRegex = /^#[0-9a-fA-F]+, #[0-9a-fA-F]+(, #[0-9a-fA-F]+)?$/;
   if (!badgeStyleRegex.test(badgeMetadata[0]?.innerText)) return null;
-  const style = badgeMetadata[0].innerText;
-  const badgeBackgroundColor = style.split(',')[0].trim();
-  const badgeColor = style.split(',')[1].trim();
+  const style = badgeMetadata[0].innerText.split(',').map((s) => s.trim());
+  if (style.length < 2) return null;
+  const badgeBackgroundColor = style[0];
+  const badgeColor = style[1];
+  const borderColor = style[2] !== 'none' ? style[2] : null;
   const badgeWrapper = badgeMetadata[0].parentNode;
   const badgeText = badgeMetadata[1].innerText;
   badgeWrapper.remove();
-  return { badgeBackgroundColor, badgeColor, badgeText };
+  return { badgeBackgroundColor, badgeColor, badgeText, borderColor };
```

Comment:
What do you think about this approach?
```suggestion
  return {style[0], style[1], badgeMetadata[1].innerText, style[2] !== 'none' ? style[2] : null };
```
We can remove the const declarations since they are not used anywhere else. On the other side, the code might get a bit hard to read, but a comment like `// badgebgcolor, badgecolor, etc` should help.

Reply:
Although it's a good suggestion to reduce the amount of code, I'd rather keep the consts to make it more readable and explicit. It will be hard to understand what is what otherwise.